client/components/add-listing/add-listing.html

```diff
@@ -25,8 +25,4 @@ <h1 id="listHead">List Your House</h1>
     <input id="pac-input" class="controls form-control input-lg" type="text" placeholder="Search Box">
     {{> googleMap name="map" options=mapOptions}}
   </div>
-<<<<<<< HEAD
-</template>
-=======
-</template>
->>>>>>> upstream/master
+</template>
```

Comment:
Do not create commits that just remove newlines. Besides files should have a newline at the end.